Server/VikingVault.Services/UserService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using VikingVault.DataAccess;
+using VikingVault.DataAccess.Models;
+using VikingVault.Services.Abstractions;
+
+namespace VikingVault.Services
+{
+    public class UserService : IUserService
+    {
+        private readonly VikingVaultDbContext _dbContext;
+
+        public UserService(VikingVaultDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+        public User CreateUser(User user)
+        {
+            try
+            {
+                _dbContext.Add(user);
+                _dbContext.SaveChanges();
+                return user;
+            }
+            catch(Exception e)
+            {
+                return null;
```

Comment:
This way of handling errors is less than optimal since we don`t have any information regarding the exception on the upper layers.